app/src/ui/stash-changes/stash-and-switch-branch-dialog.tsx

```diff
@@ -50,7 +57,10 @@ export class StashAndSwitchBranch extends React.Component<
         loading={isStashingChanges}
         disabled={isStashingChanges}
       >
-        <DialogContent>{this.renderStashActions()}</DialogContent>
+        <DialogContent className="content-wrapper">
```

Comment:
The `DialogContent` component already adds has a class, `dialog-content`. Could we drop this and use the standard `.dialog-content` selector in `_stash-changes.scss` instead?